prov/efa/src/rxr/rxr_read.c

```diff
@@ -511,8 +513,10 @@ int rxr_read_post(struct rxr_ep *ep, struct rxr_read_entry *read_entry)
 
 	peer = rxr_ep_get_peer(ep, read_entry->addr);
```

Comment:
Let's move this call to inside the `if` condition.

Reply:
Sure. Pushed new rev.